client/src/views/Simulation/components/ModelPanel.vue

```diff
@@ -81,6 +78,31 @@
       { name: capitalize(VIEWS.FUNCTIONAL), id: VIEWS.FUNCTIONAL },
     ];
 
+    onNodeClick (selected: Graph.GraphNodeInterface): void {
+      const selectedName = selected.label;
+      this.getSimParameters.forEach(parameter => {
+        if (parameter.metadata.name === selectedName) {
+          parameter.hidden = !parameter.hidden;
```

Comment:
This line 85 and line 97 are where the conflict with my PR will be, instead of `hidden` it's going to be `edited`.

Reply:
this will be dealt with in a seperate PR, changing to `edited` turned out to be larger task than thought.